client/transactions/list/index.tsx

```diff
@@ -52,6 +54,22 @@ import DownloadButton from 'components/download-button';
 import { getTransactionsCSV } from '../../data/transactions/resolvers';
 import p24BankList from '../../payment-details/payment-method/p24/bank-list';
 
+const siteLang = document.documentElement.lang;
+const siteNumberOptions = {
+	thousandSeparator: ',',
+};
+
+if ( [ 'fr-CA', 'pl-PL', 'fr-FR', 'fr-BE' ].includes( siteLang ) ) {
```

Comment:
Few notes:
- How do we know this list is inclusive? A comment on the docs might be helpful for reference.
- We could use something like `startsWith` to avoid having cases like `fr-CA` and `fr-BE` separately. If it starts with `fr` we could consider it valid for this case, similarly for others.
- Encapsulating this logic to a separate function and then calling that would read better instead of having it in the global scope at the beginning of the file.

Reply:
> How do we know this list is inclusive?

WCPay is now available in [18 countries](https://woocommerce.com/payments/#:~:text=Where%20in%20the%20world%3F,Kingdom%2C%20and%20the%20United%20States.). I got the locale for these languages from the WordPress.org [translate page](https://make.wordpress.org/polyglots/teams/). And then checked the thousand separator by changing the site language to each of these languages.